StRoot/StarGenerator/Pythia8_2_35/StarPythia8.cxx

```diff
@@ -30,7 +30,7 @@ StarPythia8::StarPythia8(const char *name) : StarGenerator(name)
   // auxilliary files required by pythia. 
   // NOTE:  When adding new versions of Pythia8, we need to specify
   // the version of the code in the source code
-  TString path = "StRoot/StarGenerator/"; path+= Pythia8_version; path+="/xmldoc/"; 
+  TString path = "StRoot/StarGenerator/"; path+= Pythia8_version; path+="/share/xmldoc/"; 
```

Comment:
Are you trying to make this relative path work for all Pythia versions in this repo?
If so, it seems like it will not work for neither Pythia8_1_xx nor Pythia8_2_35

StRoot/StarGenerator/Pythia8_1_62/xmldoc/
StRoot/StarGenerator/Pythia8_1_86/xmldoc/
StRoot/StarGenerator/Pythia8_2_35/share/Pythia8/xmldoc/